qualtran/bloqs/chemistry/trotter/inverse_sqrt_test.py

```diff
@@ -38,7 +38,7 @@ def test_poly_eval_inverse_sqrt(bloq_autotester):
 def test_newton_raphson_inverse_sqrt_bloq_counts():
     bloq = NewtonRaphsonApproxInverseSquareRoot(7, 8, 12)
     _, counts = bloq.call_graph()
-    assert counts[TGate()] == 1632
+    assert counts[TGate()] == 1424
```

Comment:
Why did this number change? Because of the bug where input bitsizes were flipped? What's the expected gate count?

Reply:
Right, I had accidentally fed the integer bitsize into the real bitsize. The cost for the scaling a real by an int is like b_r * (2 * b_i - 1) - b_i^2, so the error is whatever this is when the arguments are flipped. I added a better test which should have previously failed.